src/coreclr/interpreter/compiler.cpp

```diff
@@ -679,6 +720,22 @@ void InterpCompiler::EmitCode()
     m_methodCodeSize = (int32_t)(ip - m_pMethodCode);
 
     PatchRelocations(&relocs);
+
+    int j = 0;
+    for (int i = 0; i < m_numILVars; i++)
+    {
+        assert(m_pVars[i].ILGlobal);
+        eeVars[j].startOffset          = ConvertOffset(m_pVars[i].liveStart->nativeOffset); // This is where the variable mapping is start to become valid (inclusive?)
```

Comment:
Couldn't we just use start as 0 and end as method code size instead, rather than making use of the liveStart for these vars ?

Reply:
Should be updated now.